### ISSUE TITLE
Really old API docs are indexed by Google

### ISSUE THREAD

**Issue by mnquintana** [atom/atom.io#88]
![image](https://cloud.githubusercontent.com/assets/823545/17426279/f6e572b6-5a8a-11e6-84ee-1b85f53671bc.png)

Ideally, we should only let `atom.io/docs/latest/` URLs be indexed, instead of any corresponding to a specific version.

**Comment by Aerijo** [comment]
Yup, not good. It would also help if there was a red banner at the top of the outdated API pages saying they're outdated.

**Comment by simurai** [comment]
Maybe some sort of "if not `latest`, add `<meta name=”robots” content=”noindex”>`" might help?

> Ideally, we should only let atom.io/docs/latest/ URLs be indexed

It's probably also bad for SEO to keep changing the URL with each new version? I guess good to have a permanent link.
